src/main/java/com/exasol/adapter/document/mapping/TableMapping.java

```diff
@@ -141,6 +141,26 @@ public boolean isRootTable() {
         return this.pathInRemoteTable.size() == 0;
     }
 
+    /**
+     * Returns a string representation of the {@link TableMapping} instance.
+     * <p>
+     * This includes the Exasol table name, remote source name, column mappings,
+     * path to the nested table (if any), and any additional configuration.
+     *
+     * @return a human-readable string describing the table mapping
+     */
+    @Override
+    public String toString() {
+        return String.format(
+                "TableMapping{exasolName='%s', remoteName='%s', columns=%s, pathInRemoteTable=%s, additionalConfiguration='%s'}",
+                exasolName,
+                remoteName,
+                columns,
+                pathInRemoteTable,
+                additionalConfiguration
+        );
+    }
```

Comment:
You could add a test for `toString()` using https://github.com/jparams/to-string-verifier. Works similar to equals-verifier.